package.json

```diff
@@ -84,7 +84,7 @@
     }
   ],
   "peerDependencies": {
-    "react": ">= 18"
+    "react": ">= 17"
```

Comment:
Could consider adding a CI check that installs React 17 and runs the test suites.

Kind of a pain, but may be worthwhile if we need to support 17 and 18 in consumers.

Reply:
@ahuth do we have an example of this somewhere? I can dig in... sounds like we'd want to configure a test run for each react version and both should pass

Reply:
No examples that I'm aware of.

Would need to jump through some hoops to do it (maybe another job that runs `npm install` and then `npm install react@v17.x.x`?)

Reply:
@ahuth that makes sense. I'll create a ticket to implement this in our checks. It'll be a good one to have to prevent accidental coupling.

It's likely some other open source projects might do the same too, so I can look externally for some kinda set up.